tink/client_test/workflow_test.go

```diff
@@ -56,7 +58,14 @@ func TestWorkflowLifecycle(t *testing.T) { //nolint:paralleltest
 	}()
 
 	// Create the workflow
-	workflowID, err := workflowClient.Create(ctx, testTemplate.Id, testHardware.Id)
+	hwString := "{"
```

Comment:
Can you please elaborate in commit message how tests are failing and what fix is doing? Because it doesn't seem obvious to me.